src/components/Navbar.js

```diff
@@ -52,12 +52,17 @@ export default function Header({ children }) {
                 </Flex>
                 <HStack spacing={4} display={{ base: 'none', md: 'flex' }}>
                     <Link to="/canvasToCode">
+                        <Text fontSize="md" color={color} fontWeight={location.pathname === '/canvasToCode' ? 'bold' : 'normal'}>
+                            CanvasToCode
+                        </Text>
+                    </Link>
+                    <Link to="/wizard">
                         <Text
                             fontSize="md"
                             color={color}
-                            fontWeight={location.pathname === '/canvasToCode' || location.pathname === '/' ? 'bold' : 'normal'}
+                            fontWeight={location.pathname === '/wizard' || location.pathname === '/' ? 'bold' : 'normal'}
```

Comment:
Clicking on CanvasToCode opens up 2 options
 
Basic or Advanced
 
Selecting Basic will take the user to Wizard
 
Advanced will directly land on Canvas